src/local-ssh/ipc/extension.ts

```diff
@@ -185,22 +214,26 @@ export class ExtensionServiceServer extends Disposable {
         this.server.listen('127.0.0.1:0').then((port) => {
             this.ipcPort = port;
             this.logService.info('extension ipc service server started to listen with id: ' + this.id);
-            this.pingLocalSSHService();
-            this.backoffActive();
+            this.setExtensionActivedForDaemon();
         }).catch(e => {
             this.logService.error(e, `extension ipc service server failed to listen with id: ${this.id}`);
         });
     }
 
-    private async backoffActive() {
+    private async setExtensionActivedForDaemon() {
+        while (true) {
```

Comment:
should not we cancel such things that our extension don't keep doing anything if deactivated